yt_idv/scene_components/base_component.py

```diff
@@ -18,7 +18,9 @@
     default_shader_combos,
 )
 
-_cmaps = ["arbre", "viridis", "magma", "doom"]
+_cmaps = ["arbre", "viridis", "magma", "doom", "cividis", "plasma", "RdBu", "coolwarm"]
```

Comment:
wanna add any other [cmyt colormaps](https://github.com/yt-project/cmyt)? I like dusk and octarine :) the monochromatic ones might be nice?

if we wanted to add **all** of them, could do the following but not sure we want a matplotlib import at this level of the code?

```python
import cmyt
from matplotlib.colors import Colormap
cmyt_maps =  [cm for cm in dir(cmyt) if isinstance(getattr(cmyt, cm), Colormap)]
print(cmyt_maps)
['algae', 'algae_r', 'apricity', 'apricity_r', 'arbre', 'arbre_r', 'dusk', 'dusk_r', 'kelp', 'kelp_r', 'octarine', 'octarine_r', 'pastel', 'pastel_r', 'pixel_blue', 'pixel_blue_r', 'pixel_green', 'pixel_green_r', 'pixel_red', 'pixel_red_r', 'xray', 'xray_r']
```

Reply:
Turn this into a suggestion and I'll accept!

Reply:
oh, oops. turns out it'll be a bit longer hah... to add their names here and have it work without other modifications we'd have to add them with a `cmyt.` prefix, which will totally throw off the alphabetical sorting. i'd prefer them to be sorted by colormap name... but will need to add a bit of extra validation and touch a few more files. i'm just gonna do it as a followup PR rather than push here :)

Reply:
Cool!  Should I close this PR without merging?

Reply:
I did merge!